sdk/test/UnitTests/AWSSDK.UnitTests.NetFramework.csproj

```diff
@@ -23,7 +23,7 @@
   <Choose>
     <When Condition=" '$(AWSKeyFile)' == '' ">
       <PropertyGroup>
-        <AssemblyOriginatorKeyFile>../../../awssdk.dll.snk</AssemblyOriginatorKeyFile>
```

Comment:
This file keeps changing (the correct is the one on the right), but I found out why: In the generator, we have a hard-coded check for `net35` and `net45`: https://github.com/aws/aws-sdk-net/blob/fad56d05b452da7623071a1517378223ff38db27/generator/ServiceClientGeneratorLib/Generators/ProjectFiles/VS2017ProjectFile.tt#L133

I'll update that in a separate PR (we also have `namespace AWSSDK_DotNet35` and `namespace AWSSDK.UnitTests.Net45` hardcoded in a few places...)